doc/changelog.md

```diff
@@ -1,30 +1,201 @@
-# Thrust v1.9.8 (CUDA 11.0)
+# Thrust 1.9.10 (NVIDIA HPC SDK 20.5)
 
 ## Summary
 
-Thrust v1.9.8, which is included in the CUDA 11.0 release, removes Thrust's
-  internal derivative of CUB, upstreams all relevant changes too CUB, and adds
-  CUB as a Git submodule.
+Thrust 1.9.10 is the release accompanying the NVIDIA HPC SDK 20.5.
+It adds CMake support for compilation with NVC++ and a number of minor bug fixes
+  for NVC++.
+It also adds CMake `find_package` support.
+
+## New Features
+
+- #1130: CMake `find_package` support.
+  This is significant because there is a legacy `FindThrust.cmake` script
+    authored by a third party in widespread use in the community which has a
+    bug in how it parses Thrust version numbers which will cause it to
+    incorrectly parse 1.9.10.
+  This script only parses the first digit of each part of the Thrust version
+    number: for example, Thrust 17.17.17 would be interpreted as Thrust 1.1.1.
+  You can find directions for using the new CMake `find_package` support and
+    migrating away from the legacy `FindThrust.cmake` [here](https://github.com/thrust/thrust/blob/master/thrust/cmake/README.md)
+- #1129: Added `thrust::detail::single_device_tls_caching_allocator`, a
+    convenient way to get an MR caching allocator for device memory, which is
+    used by NVC++.
+
+## Other Enhancements
+
+- #1129: Refactored RDC handling in CMake to be a global option and not create
+    two targets for each example and test.
+
+## Bug Fixes
+
+- #1129: Fixed the legacy `thrust::return_temporary_buffer` API to support
+    passing a size.
+  This was necessary to enable usage of Thrust caching MR allocators with
+    synchronous Thrust algorithms.
+  This change has allowed NVC++’s C++17 Parallel Algorithms implementation to
+    switch to use Thrust caching MR allocators for device temporary storage,
+    which gives a 2x speedup on large multi-GPU systems such as V100 and A100
+    DGX where `cudaMalloc` is very slow.
+- #1128: Respect `CUDA_API_PER_THREAD_DEFAULT_STREAM`.
+  Thanks to Rong Ou for this contribution.
+- #1131: Fix the one-policy overload of `thrust::async::copy` to not copy the
+    policy, resolving use-afer-move issues.
+- #1145: When cleaning up type names in `unittest::base_class_name`, only call
+    `std::string::replace` if we found the substring we are looking to replace.
+- #1139: Don't use `cxx::__demangle` in NVC++.
+- #1102: Don't use `thrust::detail::normal_distribution_nvcc` for Feta because
+    it uses `erfcinv`, a non-standard function that Feta doesn't have.
+
+# Thrust 1.9.9 (CUDA Toolkit 11.0 Early Access)
```

Comment:
Should this be CUDA 11.0 GA rather than EA?

Reply:
Uh yes, yes it should.